modules/ROOT/pages/mutations/create.adoc

```diff
@@ -152,15 +153,17 @@ mutation {
 }
 ----
 
-This will ensure that a movie with ID 1234 exists and it is connected to `"Tom Hanks"`. If the movie does not exist, it will be created with the title `"Forrest Gump"`. Note that if the movie with the given ID already exists, it will be connected to it, regardless of the title.
+This ensures that a movie with ID 1234 exists and is connected to `"Tom Hanks"`.
+If the movie does not exist, it will be created with the title `"Forrest Gump"`.
+If a movie with the given ID already exists, it will also be connected to `"Tom Hanks"`, and keep whatever title it has.
 
 == `CREATE` optimization
 
-With the `create` operations, there is no limit on how many nodes can be created at once.
+With `CREATE` operations, there is no limit on how many nodes can be created at once.
```

Comment:
Nitpick: FYI  the `connectOrCreate` is camelcase but `CREATE` all uppercase idk if this is intended